examples/website/image-tile/app.tsx

```diff
@@ -37,43 +44,40 @@ export default function App({autoHighlight = true, onTilesLoad}) {
       const xmlText = await response.text();
       const dziXML = new DOMParser().parseFromString(xmlText, 'text/xml');
 
-      if (Number(dziXML.getElementsByTagName('Image')[0].attributes.Overlap.value) !== 0) {
+      if (Number(dziXML.getElementsByTagName('Image')[0].attributes.getNamedItem('Overlap')?.value) !== 0) {
         // eslint-disable-next-line no-undef, no-console
         console.warn('Overlap parameter is nonzero and should be 0');
       }
       setDimensions({
-        height: Number(dziXML.getElementsByTagName('Size')[0].attributes.Height.value),
-        width: Number(dziXML.getElementsByTagName('Size')[0].attributes.Width.value),
-        tileSize: Number(dziXML.getElementsByTagName('Image')[0].attributes.TileSize.value)
+        height: Number(dziXML.getElementsByTagName('Size')[0].attributes.getNamedItem('Height')?.value),
+        width: Number(dziXML.getElementsByTagName('Size')[0].attributes.getNamedItem('Width')?.value),
+        tileSize: Number(dziXML.getElementsByTagName('Image')[0].attributes.getNamedItem('TileSize')?.value)
       });
     };
     getMetaData();
   }, []);
 
   const tileLayer =
     dimensions &&
-    new TileLayer({
+    new TileLayer<ImageBitmap>({
       pickable: autoHighlight,
       tileSize: dimensions.tileSize,
       autoHighlight,
       highlightColor: [60, 60, 60, 100],
       minZoom: -7,
       maxZoom: 0,
-      coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
```

Comment:
Why isn't this necessary anymore?

Reply:
coordinateSystem is default to CARTESIAN when using non-geospatial viewports.